README.md

```diff
@@ -125,7 +125,10 @@ you are interested in hearing about it.
 
 ## How to use the **@coqbot** instance ##
 
-To use the bot without deploying your own instance, follow these steps:
+### As a GitHub App
+The bot can be installed as a GitHub App to either your account or
+organization (insert link to the app in the marketplace). Once you
```

Comment:
It's not listed (yet) in the marketplace but there is already a public installation link: https://github.com/apps/coqbot-app